tests/api_tests.py

```diff
@@ -162,29 +162,6 @@ def test_motor_on_for_degrees(self):
         self.assertEqual(m.speed_sp, int(round(0.75 * 1050)))
         self.assertEqual(m.position_sp, 100)
 
-        # zero speed (on-device, this will return immediately due to reported stall)
-        m.on_for_degrees(0, 100)
```

Comment:
Why are these tests being deleted?

Reply:
They are doing things like passing in speeds and degrees with values of 0 which now are caught by asserts.